### DISCUSSION TITLE
Pacstall

### DISCUSSION THREAD

**Discussion by trymeouteh** [tauri-apps/tauri#4011] [Ideas]
Please add this sdk into the pacstall repository. Pacstall is the AUR for Ubuntu which will allow Ubuntu users to easily install the sdk and receive updates very shortly after an update is released.

**Comment by FabianLars** [comment]
I don't really understand what we're supposed to add there. We don't have anything tauri-specfic in any other repositories and i can't even think of anything we could put in there 🤔
